label_studio/io_storages/azure_blob/models.py

```diff
@@ -45,21 +45,38 @@ class AzureBlobStorageMixin(models.Model):
     account_key = models.TextField(
         _('account_key'), null=True, blank=True,
         help_text='Azure Blob account key')
+    sas_token = models.TextField(
+        _('sas_token'), null=True, blank=True,
+        help_text='Azure Blob account key')
 
     def get_account_name(self):
         return str(self.account_name) if self.account_name else get_env('AZURE_BLOB_ACCOUNT_NAME')
 
     def get_account_key(self):
         return str(self.account_key) if self.account_key else get_env('AZURE_BLOB_ACCOUNT_KEY')
 
+    def get_sas_token(self):
+        return str(self.sas_token) if self.sas_token else get_env('AZURE_BLOB_SAS_TOKEN')
+
+    def get_container(self):
+        return str(self.container) if self.container else get_env('AZURE_BLOB_CONTAINER')
+
+    def get_connection_str(self, account_name, account_key, sas_token=None) -> str:
```

Comment:
we can't ignore `account_key` in the argument list although it could be ignored from an implementation perspective. Let's make 2 separate methods, one with `account_key`, another one with `sas_token`.

Reply:
I created two methods, and tested on my local.